src/strings.cpp

```diff
@@ -194,6 +194,22 @@ static bool GetSpecialNameString(StringBuilder &builder, StringID string, String
 
 static void FormatString(StringBuilder &builder, const char *str, StringParameters &args, uint case_index = 0, bool game_script = false, bool dry_run = false);
 
+/**
+ * Parse most format codes within a string and write the result to a buffer.
+ * This is a wrapper for a span of StringParameter which creates the StringParameters state and forwards to the regular call.
+ * @param builder The string builder to write the final string to.
+ * @param str Pointer to string to format.
+ * @param params The span of parameters to pass.
+ * @param case_index The current case index.
+ * @param game_script True when doing GameScript text processing.
+ * @param dry_run True when the args' type data is not yet initialized.
+ */
+static void FormatString(StringBuilder &builder, const char *str, std::span<StringParameter> params, uint case_index = 0, bool game_script = false, bool dry_run = false)
+{
+	StringParameters tmp_params = params;
```

Comment:
```suggestion
	StringParameters tmp_params{params};
```
For consistency with line 351.